app/src/ui/window/window-controls.tsx

```diff
@@ -97,8 +97,7 @@ export class WindowControls extends React.Component<{}, IWindowControlState> {
 
     return (
       <button
-        aria-label={name}
-        title={title}
```

Comment:
I think this was here for the sake of the tooltip. I am do not think we need it (atleast not from accessibility standpoint since the aria-label is there). But, open to discussion about switching this out for one our tooltips.

Other notes, the `aria-label` should be sentence case and was not, but the title was, so I swapped that.

Reply:
I'd say we should do whatever the OS does, for consistency 🤔

Reply:
Here shows before and what applying our tooltips looked like. I thought it was too blendy.. especially in contrast to what was there. So I explored tweaking the tooltip colors for this use case.

https://github.com/desktop/desktop/assets/75402236/5b300acb-eff3-441e-913e-bcbbd501a1ec

Here is what I landed on by adding some custom colors for tooltip hovering over toolbar:
(Reused the toolbar button hover css variables, but I didn't want them to be connected since they don't directly relate, so made some new variables for them)

https://github.com/desktop/desktop/assets/75402236/d637f81f-61d6-442b-83e0-09bca9d37a58